src/GraphQLServiceProvider.php

```diff
@@ -77,6 +78,8 @@ protected function bootPublishes(): void
     protected function bootTypes(): void
     {
         $configTypes = config('graphql.types');
+
+        $configTypes['Upload'] = UploadType::class;
```

Comment:
Ah, I see!

Now instead of using this:
```php
    public function args(): array
    {
        return [
            'files' => [
                'type' => Type::listOf(UploadType::getInstance()),
            ],
        ];
    }
```
we can use
```php
                'type' => Type::listOf(GraphQL::type('Upload')),
```
which also is more idiomatic how things are defined.

However my feeling is we should not add this hardcoded: not everyone might want to use the upload capability.

In fact, it's the same with the upload middleware (always enabled).

I feel there needs to be some refactoring for the middleware _and_ the `Upload` type should be added by the user manually if they want to use it (effectively making this a documentation issue).

FTR I'm also toying with the idea removing the whole upload middleware from this package in its current form and rather use https://github.com/Ecodev/graphql-upload and have an optionally enabled adapter/bridge.

Reply:
> I feel there needs to be some refactoring for the middleware and the Upload type should be added by the user manually if they want to use it (effectively making this a documentation issue).

FTR I'm talking about _outside_ this PR!

Reply:
For the UploadType change, see https://github.com/rebing/graphql-laravel/pull/401

Reply:
The UploadType change was merged, thus I've rebased this PR _and_ removed the hardcoded UploadType, as it isn't necessary anymore.